### PR TITLE
Landing page introduction

### PR DESCRIPTION
**Pull request by EdDev**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,7 +1,83 @@
-# Welcome to `nmstate`
+# We are nmstate!
+A descriptive network manager API for hosts.
```
Comment (dankenigsberg) [inline index.md]: what do you mean by "descriptive"? it is not clear to me. Maybe you'd like to sat "declarative", as it declares the requested (and current) state, not the procedure to get to it.

File: index.md [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
 
-Hello hello hello.
+## What is it?
+A library with an accompanying cli tool that manages host networking settings
```
Comment (dankenigsberg) [inline index.md]: "cli" acronym might frighten new comers. "command line" is clearer

File: index.md [hunk 3]
```diff
@@ -1,7 +1,83 @@
...
+in a descriptive manner. The networking state is described by a pre-defined
```
Comment (dankenigsberg) [inline index.md]: "descriptive" is not clear to me here, too.

File: index.md [hunk 4]
```diff
@@ -1,7 +1,83 @@
...
+schema, where reporting (current state) and changes (desired state) are both
```
Comment (dankenigsberg) [inline index.md]: I'd break the sentence. ("where") is not so clear.

The networking state is described by a pre-defined schema. Reporting of current state and changes to it (desired state) both comf*o*m to it.

File: index.md [hunk 5]
```diff
@@ -1,7 +1,83 @@
...
+confirming to it.
+
+Aimed to satisfy enterprise needs to manage host networking through a
```
Comment (dankenigsberg) [inline index.md]: Pleases use complete English sentence, including a subject (`nmstate` in this case)

File: index.md [hunk 6]
```diff
@@ -1,7 +1,83 @@
...
+### Install
+In the first few development cycles, it is recomended to install nmstate from
+sources.
+```
+git clone https://github.com/nmstate/nmstate.git
+cd nmstate
+pip install .
+```
+
+Note: For report only purposes, installing nmstate as a non-root user should
```
Comment (dankenigsberg) [inline index.md]: For report only purposes --> In order to report current state

**Review by EdDev — COMMENTED** [review]
Done

File: index.md [hunk 7]
```diff
@@ -1,7 +1,83 @@
-# Welcome to `nmstate`
+# We are nmstate!
+A declarative network manager API for hosts.
 
-Hello hello hello.
+## What is it?
+A library with an accompanying command line tool that manages host networking
```
Comment (dankenigsberg) [inline index.md]: I'd start a full sentence: 
NMState is a library...

**Review by dankenigsberg — COMMENTED** [review]
one tiny nit. other than that, approved.

please also include something similar in the in-source README.md.

**Review by dankenigsberg — APPROVED** [review]

**Pull request merged**